BlazarUI/app/scripts/components/repo-build/RepoBuildModulesTableRow.jsx

```diff
@@ -21,7 +21,15 @@ class RepoBuildModulesTableRow extends Component {
     );    
   }
 
+  isDebugMode() {
+    return window.location.href.indexOf('?debug') > -1;
```

Comment:
I know :disappointed:  I started the nice solution through the router and changed my mind, since this is a temporary change. Once we look at the auth'd user cookie, we'll probably compare to a list of users to determine if we should show the Singularity link. But thanks for pointing it out